samples/FtpServer_Files/app/application.cpp

```diff
@@ -28,11 +31,14 @@ void connectFail(const String& ssid, MacAddress bssid, WifiDisconnectReason reas
 
 void init()
 {
-	spiffs_mount(); // Mount file system, in order to work with files
-
 	Serial.begin(SERIAL_BAUD_RATE); // 115200 by default
 	Serial.systemDebugOutput(true); // Enable debug output to serial
 
+	// Mount file system, in order to work with files
```

Comment:
Maybe you can add a bit more text explaining the difference between the three mount commands.